spec/system/trn_requests_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'TRN requests', type: :system do
+  it 'completing a request' do
+    visit_home_page
+    press_start_button
+    expect_to_see_information_received_page
+    expect_page_to_include_information_received
```

Comment:
This could also be:

```suggestion
    given_i_am_on_the_home_page
    when_i_press_the_start_button
    then_i_see_the_confirmation_page
```

Last method can contain two assertions (or more, or just one, divide as you think makes sense).

There's no hard rules about this, but from [looking at how it's done in Apply](https://github.com/DFE-Digital/apply-for-teacher-training/blob/bdbcb2dec64190eb27674783654368e53a2f7946/spec/system/candidate_interface/feedback/candidate_arrives_from_find_to_provide_feedback_spec.rb#L7-L28), it works better when the method names loosely follow a "given (situation) / when (action) / then (outcome)" structure. It makes the method names read like documentation.

Reply:
👍